pyrobosim/pyrobosim/gui/world_canvas.py

```diff
@@ -171,6 +173,11 @@ def __init__(
             self.nav_animator.timeout.connect(self.nav_animation_callback)
             self.nav_animator.start(sleep_time_msec)
 
+    def toggle_collision_polygons(self):
```

Comment:
Add a docstring to this function

Also, you could consider calling `self.draw_and_sleep()` here instead of the one line in my previous comment. Either should work.